azurerm/internal/services/monitor/monitor_action_group_resource.go

```diff
@@ -739,23 +775,54 @@ func flattenMonitorActionGroupWebHookReceiver(receivers *[]insights.WebhookRecei
 	result := make([]interface{}, 0)
 	if receivers != nil {
 		for _, receiver := range *receivers {
-			val := make(map[string]interface{})
+			var useCommonAlert bool
+			var name, serviceUri string
 			if receiver.Name != nil {
-				val["name"] = *receiver.Name
+				name = *receiver.Name
 			}
 			if receiver.ServiceURI != nil {
-				val["service_uri"] = *receiver.ServiceURI
+				serviceUri = *receiver.ServiceURI
 			}
 			if receiver.UseCommonAlertSchema != nil {
-				val["use_common_alert_schema"] = *receiver.UseCommonAlertSchema
+				useCommonAlert = *receiver.UseCommonAlertSchema
 			}
 
-			result = append(result, val)
+			result = append(result, map[string]interface{}{
+				"name":                    name,
+				"service_uri":             serviceUri,
+				"use_common_alert_schema": useCommonAlert,
+				"aad_auth":                flattenMonitorActionGroupSecureWebHookReceiver(&receiver),
```

Comment:
there's no need to pass in a pointer here:

```suggestion
				"aad_auth":                flattenMonitorActionGroupSecureWebHookReceiver(receiver),
```